### PR TITLE
 improved comparison

### PR DESCRIPTION
**Pull request by plusgut**
The st.js example had not used variables, because of that I improved comparison example, to not use a result object as well. And I added a comma to the code, which made it to valid js.

### CODE REVIEW

**Comment by gliechtenstein** [comment]
thanks for the pr, appreciate it. 

The problem is i actually intended this example to show the contrasting effect between how convoluted the old way of doing things is compared to just using a single template, and your edit actually reduces this effect :)

But this brings up a good point because it means the example itself was not really adequate for what i was trying to demonstrate. Maybe we can come up with more of an edge case where it is impossible to make the old approach (the program approach) simple enough.

TLDR: this example should demonstrate how st.js makes it much simpler to do something that was much complicated before when done programmatically.

Hope this makes sense. i'll see if i can come up with a better example for this, but you are welcome to send another suggestion PR. Thank you!

**Comment by plusgut** [comment]
Sure thing, it's understandable that you want to make your library to look as good as possible.
